packages/component/src/Activity/StackedLayout.tsx

```diff
@@ -156,15 +157,17 @@ const StackedLayout: FC<StackedLayoutProps> = ({
                 fromUser={fromUser}
                 nub={showNub || (hasAvatar || hasNub ? 'hidden' : false)}
               >
-                {renderAttachment({
-                  activity,
-                  attachment: isMessage
-                    ? {
-                        content: activityDisplayText,
-                        contentType: textFormatToContentType(activity.textFormat)
-                      }
-                    : undefined
-                })}
+                <ActivityDecorator activity={activity}>
```

Comment:
Should this decorator be put inside `<Bubble>` instead?

Is it a `<ActivityBorderDecorator>`?

Should we refactor `<Bubble>...</Bubble>` into an `<BasicActivity>` component and everyone use it?

Reply:
I don't want to drill the whole activity into bubble as we already take some props from it.

Can we postpone the discussion till the public API phase? We may introduce more decorators by the time.